.github/workflows/deploy.yml

```diff
@@ -11,8 +11,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: zeebe-io/zbctl-action@v0.0.7
         with:
-          version: '1.2.1'
-          command: 'deploy src/main/resources/*.bpmn'
+          version: '8.0.4'
+          command: 'deploy resource src/main/resources/*.(bpmn|dmn)'
```

Comment:
:x: I'm 99% sure this won't work and the correct syntax would be `deploy resource src/main/resources/*.{bpmn,dmn}`
Here is how zbctl is called:
https://github.com/zeebe-io/zbctl-action/blob/main/index.js#L36

If I test this with: 
```
$ bash -c "zbctl deploy resource src/main/resources/*.(bpmn|dmn)"
bash: -c: line 1: syntax error near unexpected token `('
bash: -c: line 1: `zbctl deploy resource src/main/resources/*.(bpmn|dmn)'
```

but with the correct glob syntax it works (if we ignore the failed connection):
```

Perhaps you tested this with zsh which uses a different glob syntax?
$ bash -c "zbctl deploy resource src/main/resources/*.{bpmn,dmn}"
Error: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:26500: connect: connection refused"
```

Reply:
That's fair, I tested it with zsh